src/components/AssetMeta/AssetMeta.tsx

```diff
@@ -43,16 +43,46 @@ export interface AssetMetaStyles {
 }
 
 interface AssetMetaProps {
+  /**
+   * Asset Id
+   */
   assetId: number;
+  /**
+   * List of panes to be hidden
+   */
+  hidePanels?: AssetPanelType[];
+  /**
+   * Defines pane that will be activated once the data has been loaded
+   */
   tab?: string;
+  /**
+   * Function triggered when a user changes panes
+   */
+  onPaneChange?: (key: string) => void;
+  /**
+   * Object passed as props to inner component that presents details pane
+   */
+  detailsProps?: MetaDescriptionListProps;
```

Comment:
For some reason `detailsProps` as well as `timeseriesProps`, `eventProps` and `customSpinner` shows as `any` type, which is confusing. Could you check the what is a reason of this?

Reply:
@mrtraser Can we add this to #553 ?

Reply:
yes